index.rst

```diff
@@ -5,10 +5,17 @@ Welcome to the official documentation of the UBports project!
 
 UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
 
-On this website you find instructions how to install Ubuntu Touch on your mobile phone, user guides and detailed documentation on all system components. If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
+Documentation creation is being provided by Read the Docs (https://readthedocs.org/ ) which is an open source (MIT-licensed) project.  The guide is divided into general chapters to make reading easier. Docs are rendered as HTML and PDF and full-text search is available.  If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
```

Comment:
You're repeating content that's already covered on about/introduction. If you want to move/remove/replace a page, please do it in one pull request, because we won't merge unfinished pages in master.